modules/storages/app/components/storages/admin/storage_list_component.sass

```diff
@@ -1,10 +1,34 @@
 .op-storage-list--header,
 .op-storage-list--row
   display: grid
-  grid-template-columns: 1fr 180px 160px 240px
-  grid-template-areas: "name user provider time" "host user provider time"
+  grid-template-columns: 1fr 160px 180px 220px
+  grid-template-areas: "name provider user  time" "host provider user  time"
 
-@media screen and (max-width: $breakpoint-md)
+.op-storage-list--header--name
+  min-width: 140px
```

Comment:
The `min-width` can be replaced by using `minmax(140px, 1fr)` in the grid definition